drivers/sensor/tdk/icm456xx/Kconfig

```diff
@@ -0,0 +1,68 @@
+# ICM45686-P ICM45686-S Six-Axis Motion Tracking device configuration options
+#
+# Copyright (c) 2024 TDK Invensense
+# Copyright (c) 2022 Esco Medical ApS
+# Copyright (c) 2020 TDK Invensense
+#
+# SPDX-License-Identifier: Apache-2.0
+config TDK_APEX
+	bool
+
+menuconfig ICM456XX
+	bool "ICM45686 Six-Axis Motion Tracking Device"
+	default y
+	depends on ZEPHYR_HAL_TDK_MODULE
+	select SPI if $(dt_compat_on_bus,$(DT_COMPAT_INVENSENSE_ICM45686),spi)
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_INVENSENSE_ICM45686),i2c)
+	select TDK_APEX if $(dt_node_str_prop_equals,$(dt_nodelabel_path,icm45686),apex,pedometer) \
+		|| $(dt_node_str_prop_equals,$(dt_nodelabel_path,icm45686),apex,tilt) \
+		|| $(dt_node_str_prop_equals,$(dt_nodelabel_path,icm45686),apex,smd) \
+		|| $(dt_node_str_prop_equals,$(dt_nodelabel_path,icm45686),apex,wom)
+	select USE_EMD_ICM45686
+	select SENSOR_ASYNC_API
+	help
+	  Enable driver for ICM42x70 SPI-based or I2C-based Six-Axis or Three-Axis Motion Tracking device.
```

Comment:
Please update the help message. This is a leftover from icm42x70.

Reply:
Fixed